src/Compatibility/Microsoft.DotNet.ApiCompatibility/DiagnosticIds.cs

```diff
@@ -19,6 +19,8 @@ public static class DiagnosticIds
         public const string CannotSealType = "CP0009";
         public const string EnumTypesMustMatch = "CP0010";
         public const string EnumValuesMustMatch = "CP0011";
+        public const string CannotRemoveVirtualModifier = "CP0012";
```

Comment:
NIT: In order to be more consistent to other DiagnosticIds, can we name them something along the lines of `CannotMakeMemberVirtual` and `CannotRemoveVirtualFromMember`?